package.json

```diff
@@ -315,6 +315,7 @@
         "formik": "^2.0.0",
         "ilib-tree-node": "^1.2.0",
         "immutable": "^3.7.4",
+        "intl-messageformat": "^7.8.4",
```

Comment:
This commit will need to be marked as a breaking change.

Reply:
We already depend on intl-messageformat implicitly, as it is the basis for react-intl. However, lint complained about it because we didn't explicitly mention it in the package.json file when we explictly import it in the new code. So, it's not really a dependency change.